tests/Feature/NotesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Tests\TestCase;
+
+class NotesTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     */
+    public function testExample(): void
+    {
+        $response = $this->get("/");
+
+        $response->assertStatus(200);
+    }
+
+    public function testNotesAreStoredInDatabase(): void
```

Comment:
Please write more test 
- some happy path - for example, that user can see his notes, he can deletes his note,
- some unhappy path - for example that user can't create empty note and so on